src/Microsoft.AspNetCore.Mvc.Core/Routing/UrlHelper.cs

```diff
@@ -41,10 +41,19 @@ public UrlHelper(ActionContext actionContext)
         /// <inheritdoc />
         public ActionContext ActionContext { get; }
 
+        /// <summary>
+        /// Gets the <see cref="RouteValueDictionary"/>.
+        /// </summary>
```

Comment:
We might want to say more about this like "these are the route values generated from the current request, they will be used to <blah.. blah.. blah>"